packages/admin-stories/src/admin/error-handling/ErrorBoundary.tsx

```diff
@@ -0,0 +1,55 @@
+import { ErrorBoundary } from "@comet/admin";
+import { Link, Paper, Typography } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { boolean } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+const ViewWithNoError: React.FunctionComponent = () => {
+    return (
+        <div>
+            <Typography>View with No Error</Typography>
+            <Typography> Go to knobs and try to render a view with an error</Typography>
+        </div>
+    );
+};
+
+const ViewWithError: React.FunctionComponent = () => {
+    const potentialError = null;
+
+    //@ts-ignore This should really throw an error ;-)
+    potentialError.produceError();
```

Comment:
Why not simply `throw new Error(...)` 🤔

Reply:
thanks for the good input, added suggestions

Reply:
![Screenshot 2021-03-25 at 10 14 23](https://user-images.githubusercontent.com/6098356/112448644-e85a5180-8d52-11eb-82af-e5ce8cc5d181.png)

Reply:
Please add similar alerts for the other two use cases as well. This should be sufficient as documentation, in my opinion.

Reply:
most of the information was already present in the stories. I just refactored the structure of the presentation.

Reply:
fyi: 
![Screenshot 2021-03-25 at 10 59 06](https://user-images.githubusercontent.com/6098356/112454628-2d818200-8d59-11eb-94b7-92866e7e0ec7.png)
![Screenshot 2021-03-25 at 10 59 03](https://user-images.githubusercontent.com/6098356/112454637-2f4b4580-8d59-11eb-98ee-774de1e8745f.png)
![Screenshot 2021-03-25 at 10 58 58](https://user-images.githubusercontent.com/6098356/112454641-2fe3dc00-8d59-11eb-9c88-72a581f332e5.png)